clusterdock/topologies/cdh/actions.py

```diff
@@ -215,3 +281,110 @@ def remove_files(cluster, files, nodes):
     logger.info("Removing files (%s) from hosts (%s)...",
                 ', '.join(files), ', '.join([node.fqdn for node in nodes]))
     cluster.ssh('rm -rf {0}'.format(' '.join(files)), nodes=nodes)
+
+def _install_kdc_server(cluster, kdc_node, network, kerberos_principals):
+    logger.info('Installing kdc server on kdc node...')
+    _change_config_files(kdc_node, network)
+    _setup_kdc_server(cluster, [kdc_node], network.upper(), kerberos_principals)
+
+def _change_config_files(kdc_node, network):
+    logger.info('Changing config. files on kdc node...')
+
+    client = Client()
```

Comment:
Perhaps just change my import at the top to be `import docker` and use `docker.Client` here to make it clear what this `Client` is referring to.